packages/rum/README.md

```diff
@@ -272,17 +275,17 @@ datadogRum.init({
 });
 ```
 
-For example, to collect 100% of your sessions using only the Session Replay RUM option without recording a replay:
+For example, to collect 100% of your sessions with Session Replay RUM data:
 
 ```
 datadogRum.init({
     ....
     sampleRate: 100,
-    replaySampleRate: 100 // Note: if this is not included it will default to 100%
+    replaySampleRate: 100
 });
 ```
 
-For example, to collect only 50% of your overall sessions, with 25% using the Browser RUM option and the other 25% using the Session Replay RUM option, set the `sampleRate` and the `replaySampleRate` to 50.:
+For example, to collect only 50% of your overall sessions, with half collecting Browser RUM data and the other half collecting Session Replay RUM data, set the `sampleRate` and the `replaySampleRate` to 50.:
```

Comment:
```suggestion
For example, to collect only 50% of your overall sessions, with half collecting Browser RUM data and the other half collecting Session Replay RUM data, set the `sampleRate` and the `replaySampleRate` to 50.
```

Reply:
I removed the dot instead, because it is the description of the code example below